python/ray/data/_internal/planner/exchange/sort_task_spec.py

```diff
@@ -81,8 +81,9 @@ def validate_schema(self, schema: Optional[Union[type, "pyarrow.lib.Schema"]]):
             for column in self._columns:
                 if column not in schema_names_set:
                     raise ValueError(
-                        "The column '{}' does not exist in the "
-                        "schema '{}'.".format(column, schema)
```

Comment:
I think it's better to print full schema (though fixing the formatting)

Reply:
Fair enough, it will look fine when printed but when embedded it's gonna look messy.

Let's keep just the list of cols